projects/mercury/package.json

```diff
@@ -4,11 +4,12 @@
   "private": true,
   "scripts": {
     "dev": "yarn build && concurrently \"yarn localdevelopment\" \"yarn saturn\" \"yarn pluto\" \"yarn start\"",
+    "dev-no-saturn": "yarn build && concurrently \"yarn localdevelopment\" \"yarn pluto\" \"yarn start\"",
```

Comment:
Why is it needed? It looks like we started adding all possible combinations here... I would rather keep the script options simple (I already didn't like the dev-be and dev-fe separation)